### ISSUE TITLE
Crashing on Mac OSX Yosemite 10.10.5

### ISSUE THREAD

**Issue by ucojsh30** [ttscoff/nv#387]
Hello Support team,

The app no longer works, it use to crash occasionally a reboot to computer would fix but now it crashes when I type anything or do anything. I use this for business purposes and a huge blow that the update broke the app. 

Please fix.

Josh

**Comment by diffsky** [comment]
https://github.com/ttscoff/nv/issues/385#issuecomment-250498990 fixed the crashes for me

**Comment by elasticthreads** [comment]
Yes, the most recent update was released yesterday, September 30. Download it at the link below as auto-updating might not work between some earlier releases and this one (though it should work going forward):
http://brettterpstra.com/projects/nvalt/

The newest version is: 2.2 (122)

**Comment by Zortrium** [comment]
Build 122 has *not* fixed the incessant crashing issues for me.  Still crashes immediately upon opening every time.  Running El Capitan 10.11.6.

Not great that I have to keep reverting to the beta version to actually use the app...

**Comment by ttscoff** [comment]
@Zortrium you should probably include crash reports, given it seems fine for everyone else. After you trash your prefs and reboot, of course.

**Comment by elasticthreads** [comment]
@Zortrium since we don't have your crash reports I can't be sure, but the updates we released this week may have fixed your bugs. Please report back if they haven't.

**Comment by Zortrium** [comment]
I haven’t gotten any crashes recently, so I think my issues were fixed.  Thanks!
